tools/pkg/scripts/smoke_test.sh

```diff
@@ -13,10 +22,18 @@ echo "Check, that bootstrap01-hello.sh script is executed"
 BOOTSTRAP_RESULT=$(mongooseimctl bootstrap)
 echo "$BOOTSTRAP_RESULT" | grep "Hello from"
 
-echo "Check, that print_install_dir works"
-MIM_DIR=$(mongooseimctl print_install_dir)
-test -d "$MIM_DIR"
+# Script should be accessable by "mongooseim" user
```

Comment:
Can we change the owner of the moved `smoke_templates.escript` to `mongooseim`? The package is installed at this point so the user is already created if I'm correct.

Reply:
mongooseim cannot access files inside root's directory.